pkg/utils/k8sutil.go

```diff
@@ -19,19 +19,18 @@ package utils
 import (
 	"encoding/json"
 	"fmt"
-	"net/url"
-	"os"
-	"path/filepath"
-
 	httptriggerapi "github.com/kubeless/http-trigger/pkg/apis/kubeless/v1beta1"
 	kubelessApi "github.com/kubeless/http-trigger/pkg/apis/kubeless/v1beta1"
+	"github.com/sirupsen/logrus"
 	"k8s.io/api/extensions/v1beta1"
 	k8sErrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/clientcmd"
-
-	"github.com/sirupsen/logrus"
+	"net/url"
```

Comment:
the core imports should be in a block separated by a empty line from other libraries